backend/sftp/sftp.go

```diff
@@ -164,6 +164,16 @@ Home directory can be found in a shared folder called "home"
 			Default:  false,
 			Help:     "Set to skip any symlinks and any other non regular files.",
 			Advanced: true,
+		}, {
+			Name:     "subsystem",
+			Default:  "sftp",
+			Help:     "Specifies the SSH2 subsystem on the remote host.",
+			Advanced: true,
+		}, {
+			Name:     "sftp_server",
+			Default:  "",
+			Help:     "Specifies the path or command to run a sftp server on the remote host. The subsystem option is ignored when sftp_server is defined.",
```

Comment:
Can you break this line into two parts - the first line makes the option help text for `--sftp-server` and its too long!

`Specifies the path or command to run a sftp server on the remote host.

The subsystem option is ignored when sftp_server is defined.`